splight_lib/conftest.py

```diff
@@ -1,10 +1,44 @@
+from pathlib import Path
```

Comment:
@marianogaragiola I have an alternate manifest.json approach for declaring files